sprint 7/tema 3.js

```diff
@@ -0,0 +1,36 @@
+function dayOfTheWeek(weekDay){
+  
+  let dayName = parseInt(weekDay);
+
+  switch(dayName){
+    case 1:
+      console.log("Monday");
+      break;
+    case 2:
+      console.log("Tuesday");
+      break;
+    case 3:
+      console.log("Wednesday");
+      break;
+    case 4:
+      console.log("Thursday");
+      break;
+    case 5:
+      console.log("Friday");
+      break;
+    case 6:
+      console.log("Saturday");
+      break;
+    case 7:
+      console.log("Sunday");
+      break;
+    default:
+      console.log("Error");
+      break;
+  }
+
+}
+  
+  // dayOfTheWeek(7)
+  // dayOfTheWeek(1)
+  dayOfTheWeek(8)
```

Comment:
```suggestion
  dayOfTheWeek(8);
```
(optional)

Reply:
(optional) se poate adauga un text in console.log() cu explicatie ce se intampla